src/devices/Max7219/Max7219.cs

```diff
@@ -0,0 +1,323 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Device.Gpio;
```

Comment:
Correct me if I'm wrong but I don't believe you use GPIO at all here right? It looks like everything is done through SPI.